examples/seat-adjuster/requirements.in

```diff
@@ -1,4 +1,4 @@
-grpcio
+grpcio>=1.56.0
 protobuf
-dapr
+dapr>=1.10.0
```

Comment:
is that really needed? we had so many problems in the past with newer dapr versions... maybe it is better to stay on a working version as long as possible